foundationdb/tests/atomic.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018 foundationdb-rs developers, https://github.com/bluejekyll/foundationdb-rs/graphs/contributors
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+extern crate byteorder;
+extern crate foundationdb;
+extern crate foundationdb_sys;
+extern crate futures;
+
+use foundationdb::*;
+
+use byteorder::ByteOrder;
+use futures::future::*;
+
+use error::FdbError;
+
+//TODO: impl Future
+fn atomic_add(db: Database, key: &[u8], value: i64) -> Box<Future<Item = (), Error = FdbError>> {
```

Comment:
is there a big benefit for exposing individual Atomic operations, as opposed to a generic interface that calls through to atomic_op?

Reply:
I wrote it to simplify main test flow. For `Transaction::atomic_add`, I think it might worth to have because without it users should add `byteorder` as a dependency and write severeal lines of code for one atomic add operations, as this `atomic_add` does [1].

[1] https://github.com/bluejekyll/foundationdb-rs/pull/26/files/c400b23781408214ce8248c2320d47e21a05be3f#diff-54a7af36c834a0fb789fab78a0dce2f6R27